references/optical_flow/train.py

```diff
@@ -150,7 +150,7 @@ def preprocessing(img1, img2, flow, valid_flow_mask):
 
     for name in val_datasets:
         if name == "kitti":
-            # Kitti has different image sizes so we need to individually pad them, we can't batch.
+            # Kitti has different image sizes, so we need to individually pad them, we can't batch.
```

Comment:
I assume this comes from a hard rule that "so" must be preceded by a comma?
This doesn't work too well in practice IMO - but not worth nitpicking.

(I'm biased on this specific example because I think I was the one writing this comment)

Reply:
I'm by no means an expert on the English language and thus I trusted the grammar checker here. But indeed, it told me

> that "so" must be preceded by a comma

I don't know if this is just hard-coded or if this was a specific hint for the case at hand and in other situations the hint is not displayed.